src/ocean/sys/ErrnoException.d

```diff
@@ -37,12 +37,11 @@ version (UnitTest)
 
 public class ErrnoException : Exception
 {
+    import ocean.core.Exception : ReusableExceptionImplementation;
```

Comment:
:(
I feel that the reason you're having the conflicts is that imports get added randomly, instead of a defined order. But will do.